app-hw-37/src/hw37/Title/Title.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import './Title.css';
 
-export const Title = () => (<h1 className='title'>Sign In</h1>)
+export const Title = (text: { children: string }) => (<h1 className='title'>{text.children}</h1>)
```

Comment:
usually, you create a separate prop for text nodes
never see the such way. But I'd like it)